src/v3/src/components/Widget/index.tsx

```diff
@@ -304,9 +305,9 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note: we need to keep both themeproviders (MUI/ODS) until ODS exports all MUI components */}
       <MuiThemeProvider theme={brandedTheme}>
```

Comment:
This is still causing some CSP issues. The theme provider pulls `cspNonce` only from `window.cspNonce` and does not directly accept it. This means in the playground we will still see the CSP failures but in the Okta-hosted environment it will pick up the nonce and work correctly.

Ultimately, we don't need to worry about this because we are planning to retire all usage of the older Odyssey theme provider so this will get removed soon.